src/providers/postgres/qgspostgrestransaction.h

```diff
@@ -29,7 +29,17 @@ class QgsPostgresTransaction : public QgsTransaction
 
   public:
     explicit QgsPostgresTransaction( const QString &connString );
-    bool executeSql( const QString &sql, QString &error, bool isDirty = false ) override;
+
+    /**
+     * Executes the SQL query in database.
+     *
+     * \param sql The SQL query to execute
+     * \param error The error or an empty string if none
+     * \param isDirty True to add an undo/redo command in the edition buffer, false otherwise
+     * \param name Name of the transaction ( only used if `isDirty` is true)
```

Comment:
Is transaction the right term here? I think it should be "operation" or similar.

Reply:
You're right. I updated the documentation accordingly.